tests/upload_notes/test_upload_labelstudio.py

```diff
@@ -85,7 +87,6 @@ def test_basic_push(self):
                             # as it was not in our initial CUI mapping (in push_tasks)
                             {
                                 "from_name": "mylabel",
-                                "id": "ctakes0",
```

Comment:
I realized that `id` isn't a real field in this part, afaict - I don't know why it was added in the first place. But removing it lets me not keep track of a counter or what type of object is being serialized, so it's gone baby.